spec/system/home_spec.rb

```diff
@@ -174,4 +174,22 @@ def ie_alert_box_xpath
 
     expect(page).not_to have_css(".title", text: "Featured")
   end
+
+  describe "Header Card" do
+    scenario "if there is header card with link, the link content is rendered" do
+      create(:widget_card, :header, link_text: "Link text", link_url: "consul.dev")
+
+      visit root_path
+
+      expect(page).to have_css(".link-content", text: "Link text")
+    end
+
+    scenario "if there is header card without link, the link content is not rendered" do
+      create(:widget_card, :header, link_text: nil, link_url: nil)
+
+      visit root_path
+
+      expect(page).not_to have_css(".link-content")
```

Comment:
```suggestion
      within(".header-card") { expect(page).not_to have_link }
```

This requires having to add this class somewhere... maybe to the `jumbo highlight` element?